recipe-server/normandy/recipes/tests/test_api.py

```diff
@@ -272,7 +272,7 @@ def test_it_can_enable_recipes(self, api_client):
         recipe = RecipeFactory(enabled=False, approver=UserFactory())
 
         res = api_client.post('/api/v1/recipe/%s/enable/' % recipe.id)
-        assert res.status_code == 204
+        assert res.status_code == 200
```

Comment:
We should be testing for the content changes to this API as well as the status code change.